data/releasenotes.json

```
@@ -1,4 +1,22 @@
 [
+	{
+		"version": "v2025-12-02",
+		"date": "2025-12-02",
+		"breaksdb": false,
+		"notes": [
+			"<b>New Feature:</b> Support ALTER OWNER for views",
+			"<b>Performance:</b> Improve JSONB object access performance by up to 20%",
+			"<b>PostgreSQL Compatibility:</b> Respect default escape character <tt>\\</tt> for <tt>COPY FROM STDIN</tt> statements",
+			"<b>PostgreSQL Compatibility:</b> support casts from jsonb to numeric and boolean",
+			"<b>PostgreSQL Compatibility:</b> Add support for <tt>WITH ORDINALITY</tt> for <tt>json(b)_array_elements</tt> functions.",
+			"<b>Maintenance:</b> Ignore misformed values, such as invalid JSON documents, when evaluating samples for query planning.",
+			"<b>Maintenance:</b> Vacuum now removes compressed blocks",
```

Sounds like a bad idea otherwise :sweat_smile: 
```suggestion
			"<b>Maintenance:</b> Vacuum now removes no longer visible compressed blocks",
```